tools/clang/tools/dxcompiler/dxcompilerobj.cpp

```diff
@@ -1039,7 +1039,8 @@ class DxcCompiler : public IDxcCompiler3,
 
           dxcutil::AssembleInputs inputs(
               std::move(serializeModule), pOutputBlob, m_pMalloc,
-              SerializeFlags, pOutputStream, opts.GetPDBName(),
+              SerializeFlags, pOutputStream,
+              opts.SkipHash ? DxcValidatorFlags_SkipHash : 0, opts.GetPDBName(),
```

Comment:
If we want to maintain previous behavior, we may want to set the `skip_hash` flag when the internal validator is explicitly selected. I don't know why anyone would object to an unrequested hash, but the inclusion of the skip_hash flag suggests it's at least a possibility.

Reply:
The idea here is that external validator will be the same as internal validator.
The only difference is that external validator is in dxil.dll which is smaller.
Then when user use dxc, there will be no different behavior when dxil.dll exist or not.

Maybe we don't need to have the concept of internal or external validator for dxc.
The external validator is only for things other than dxc to use?

Reply:
As discussed, we may need to consider this more. In the meaintime, I think this is fine.